apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -118,6 +118,9 @@ type InputSpec struct {
 	// +optional
 	// +docgen:ignore
 	Audit *Audit `json:"audit,omitempty"`
+
+	// Source to receive logs from non-cluster sources.
+	Source *SourceSpec `json:"source,omitempty"`
```

Comment:
Is there an alternate name we can use, maybe "server" to make it more obvious the input is a receiver.   Source and Input are basically the same and the others denote essentially a well defined category of inputs (or sources)

Reply:
I have many [comments](https://github.com/openshift/cluster-logging-operator/pull/1958) regarding the API that should be addressed IMO before this PR merges.  I will skip additional  API review

Reply:
Replied on https://github.com/openshift/cluster-logging-operator/pull/1958 but I've closed that PR as it's not going to merge separately and this PR now has the "live" API design. We can continue discussion here.

Reply:
Your other comment you suggested "receiver" which I think may be a more intuitive option.  "source" is very vector centric but may man comment is the others are already "sources"; they are specific sources so why is this new one not specific too?  It specifically "receives" or acts as a server?

Reply:
+1 lets change this.